js/index.js

```diff
@@ -22,6 +22,57 @@ for (let i = 0; i < skills.length; i++) {
     skill.innerHTML = skills[i];
     skillsList.appendChild(skill);
 }
+// 4.3 Form Basics
+//*******************************************************************************************
+// create messages
+// *******************************************************************************************
+// show the information from the form
+//--------------------------------------------------------------------------------------------
+const messageForm = document.getElementsByName("leave_message");
+console.log(messageForm[0], "Show the data from the form ");
 
+messageForm[0].addEventListener("submit", function(event){
+    event.preventDefault();
 
+const theMesssageForm = event.target;
 
+var name = event.target.name.value;
+var email = event.target.email.value;
+var message = event.target.message.value;
+
+console.log("Display your name: ", name);
+console.log("Display Email: ", email);
+console.log("Display text here: ", message);
+
+// Clear the form using the reset() method 
+document.getElementById("form").reset(); 
+
+var messageSection = document.getElementById("messages");
+var messageList = messageSection.querySelector("ul");
+var newMessage = document.createElement("li");    
+
+//adding messages to the list
+messageList.appendChild(newMessage);
+
+    newMessage.innerHTML = `
```

Comment:
```suggestion
newMessage.innerHTML = `
```

This is not inside a loop or an if statement, so no reason to indent it.